.github/workflows/test.yml

```diff
@@ -103,10 +124,20 @@ jobs:
         PG_DATABASE: a12nserver
         PG_HOST: 127.0.0.1
       run: make knex-migrate
+    - name: Test seed migrations
+      env:
```

Comment:
I noticed these environment variables are duplicated. Is there a way these can be specified in a way that they are re-used across steps? Maybe `env` can be specified on the job level?

Reply:
I think they might be needed on every one of these, seems like I borked it by refactoring them onto the parent job